### PR TITLE
LPS-135282 Navigation bar overlaps the user dropdown menu

### PR DESCRIPTION
**Pull request by mateusgomes01**

### CODE REVIEW

**Comment by mateusgomes01** [comment]
ci:test:sf

ci:test:relevant

**Comment by mateusgomes01** [comment]
ci:test:sf

ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -29,7 +29,7 @@
 	min-width: 100%;
 }
 
-.dropdown-menu.show {
+.ddm-field-dropdown.show {
```
Comment (miltonmc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/main.scss:32]: I didn't get the reason for the class name change

**Comment by mateusgomes01** [comment]
Closing. Will send another PR

**Pull request closed without merging**